demisto_sdk/commands/common/errors.py

```diff
@@ -1541,6 +1554,15 @@ def invalid_incident_field_in_layout(invalid_inc_fields_list):
                "2 - The id_set.json file is up to date. Delete the file by running: rm -rf Tests/id_set.json and" \
                " rerun the command."
 
+    @staticmethod
+    @error_code_decorator
+    def invalid_incident_field_in_pre_process_rules(invalid_inc_fields_list):
```

Comment:
need to check for the script existence as well

Reply:
https://github.com/demisto/demisto-sdk/commit/4381614eade432d3ef64e3b0890fe01ab03d6a65